Formula/beagle.rb

```diff
@@ -0,0 +1,28 @@
+class Beagle < Formula
+  desc "Evaluate the likelihood of sequence evolution on trees"
+  homepage "https://github.com/beagle-dev/beagle-lib"
+  url "https://github.com/beagle-dev/beagle-lib/archive/v3.0.2.tar.gz"
+  sha256 "59a7081b61ead0a5738e813c6fcfb614d2c5bb49c29b28609de8e2b51bea3ec0"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "doxygen" => :build
+  depends_on :java => :build
```

Comment:
really only build-time?

Reply:
Yup, `beagle` builds Java bindings to its native library, but not all downstream apps need it. See [mrbayes](https://github.com/brewsci/homebrew-bio/blob/master/Formula/mrbayes.rb). Haven't figured out a way to disable it and `beast` needs the Java bindings anyway which is why we're migrating it in the first place...

Reply:
maybe `=> [:build, :test]` then and add a test for the Java bindings? (see Octave)